clayui.com/src/templates/docs.js

```diff
@@ -242,12 +249,51 @@ export default props => {
 												{hasTabs && (
```

Comment:
Rather than using the `has*` variables, we could have something like this for the render logic

```jsx
{tab && (
	<>
		{tab.html && (
			<div
				dangerouslySetInnerHTML={{
					__html:
						tab.html,
				}}
			/>
		)}

		{tab.code && (
			// MDX render logic
		)}
	</>
)}
```